test/models/023-item.coffee

```diff
@@ -140,17 +123,3 @@ describe 'item model', ->
       updateAttributesData = { listing: 'chocolat' }
       (-> update(updateAttributesData, doc)).should.throw()
       done()
-
-    it 'should not throw when updated with a valid snapshot', (done)->
-      doc = create validItem
-      updateAttributesData =
-        snapshot:
-          'entity:image': '/ipfs/QmXtPA8rAn5VCJ3rVLxWhDEnMv2hiPTMopH9NuaERz3nuw'
-      (-> update(updateAttributesData, doc)).should.not.throw()
-      done()
-
-    it 'should throw when updated with an invalid snapshot', (done)->
-      doc = create validItem
-      updateAttributesData = { snapshot: { bla: 'hello' } }
-      (-> update(updateAttributesData, doc)).should.throw()
```

Comment:
is this deletion tested somewhere else now. maybe in `/api_tests/items/snapshot.test.coffee` ?

Reply:
is it fine without testing ? :blush:

Reply:
now that the snapshot aren't written in items documents anymore, the closest possible would be to test the schema of snapshot data attached to items, [which is done here](https://github.com/inventaire/inventaire/blob/29fd78c/tests/api/items/snapshot.test.coffee). so I think it's fine without additional test